systemd/ocp-cluster-ca.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# To generate the custom-ca.crt
+# USER="system:admin"
+# GROUP="system:masters"
+# USER_SUBJ="/O=${GROUP}/CN=${USER}"
+# NAME="custom"
+# CA_SUBJ="/OU=openshift/CN=admin-kubeconfig-signer-custom"
+# VALIDITY=3650
+
+# openssl genrsa -out $NAME-ca.key 4096
+# openssl req -x509 -new -nodes -key $NAME-ca.key -sha256 -days $VALIDITY -out $NAME-ca.crt -subj "$CA_SUBJ"
+# openssl req -nodes -newkey rsa:2048 -keyout $USER.key -subj "$USER_SUBJ" -out $USER.csr
+# openssl x509 -extfile <(printf "extendedKeyUsage = clientAuth") -req -in $USER.csr \
+#    -CA $NAME-ca.crt -CAkey $NAME-ca.key -CAcreateserial -out $USER.crt -days $VALIDITY -sha256
+
+set -x
+
+if [ -z $CRC_CLOUD ]; then
+    exit 1
+fi
```

Comment:
What is this condition for and from where `CRC_CLOUD` env will come from?

Reply:
it is set by the systemd unit based on an env file, if you check the `ocp-cluster-ca.service` file there's a 
```
EnvironmentFile=/opt/crc/crc-cloud
```

so when we create the AMI or an disk image for a different provider, we'll create this environment file during that generation of the cloud images